### ISSUE TITLE
Form elements

### ISSUE THREAD

**Issue by jonnyry** [nhsconnect/gpconnect-demonstrator#346]
All form elements should have descriptive text associated to them. This will help users of assistive technology determine their function or purpose.
As these are icon buttons, visually hidden text can be used to inform screen reader users what the buttons are for.

E.g.
```
<button type="button" class="btn btn-default btn-sm" ng-click="openDatePicker($event, 'fromDate')">
<i class="fa fa-calendar"></i>
<span class="">open from date picker</span> </button>
```

**Comment by jonnyry** [comment]
- [x] "Open from date picker" and "Open to date picker" buttons on 7/10 HTML view pages.
- [ ] Advanced patient search "open date picker"
- [ ] Date picker left and right buttons
- [ ] Info button
- [ ] Back button
